functions.lua

```diff
@@ -47,6 +47,11 @@ function wrench.pickup_node(pos, player)
 			return false, errors.owned:format(owner)
 		end
 	end
+	local drop = def.drop and minetest.registered_nodes[node.name].drop
+	if drop then
+		-- drop node name like digging
+		node.name = drop
+	end
```

Comment:
`drop` can also be a table: [lua_api.txt#L7861-L7914](https://github.com/minetest/minetest/blob/80c3c7e642749f9316a3eee2c235df3ce8be1666/doc/lua_api.txt#L7861-L7914).

I think it would be better to specify it in the wrench definition.
```suggestion
	if def.pickup_name then
		node.name = def.pickup_name
	end
```